mplex/src/lib.rs

```diff
@@ -453,7 +473,14 @@ mod tests {
         let mut inbound: Vec<Substream<_>> = vec![];
 
         for _ in 0..5 {
-            inbound.push(mplex.clone().inbound().wait().unwrap());
+            inbound.push(
+                mplex
+                    .clone()
+                    .inbound()
+                    .wait()
+                    .unwrap()
+                    .expect("inbound substream"),
```

Comment:
This isn't much of a problem but can you try to make expect messages more helpful in the future? If nothing else, helpful error messages are really useful for other developers (for example, future-Pierre) to reverse-engineer your thinking as to why this can't fail in the first place.

Reply:
In general I stay clear of `unwrap`/`expect` altogether with tests being the only exception. Normally I would just have used `unwrap` here, but to better distinguish it from the other `unwrap` that was already there I went with `expect` instead.